### PR TITLE
[community-4.10] Remove subscription requirement

### PR DESCRIPTION
**Pull request by alinaryan**
Removes the subscription requirement from the description as it
is not mandatory for community WMCO. 

Corrects doc links.

### CODE REVIEW

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -21,126 +21,31 @@ spec:
...
-    - [AWS](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md)
-    - [Azure](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-azure.md)
+    Please see the usage section of [README.md](https://github.com/openshift/windows-machine-config-operator/blob/community-4.10/README.md#usage).
 
     ### Limitations
     #### DeploymentConfigs
     Windows Nodes do not support workloads created via DeploymentConfigs. Please use a normal Deployment, or other method to
     deploy workloads.
-
-    ### Reporting issues
```
Comment (saifshaikh48) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:140]: Should we point people to our github instead?

**Review by saifshaikh48 — COMMENTED** [review]
lgtm aside from one question

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (alinaryan) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:140]: It's mentioned on [line 27](https://github.com/openshift/windows-machine-config-operator/pull/1161/commits/e4b4c08a71ae3092e7ea4468444a5348db8cfdb8#diff-53f141ca70f6ce0a959fa80b57ab2ac4e1e8f29689213aebce7adf958f5c71caR27)

**Comment by sebsoto** [comment]
I'm confused as to why this change needs to be made. I thought the community CSV description lives in the [description.md](https://github.com/openshift/windows-machine-config-operator/blob/master/hack/community/csv/description.md) file you added?

Making the changes present in this PR, specifically to the community branch, means we'll have to do this every time a community branch is created. Thats one of the problems thats solved by the generate.sh script and the above linked file.

**Comment by alinaryan** [comment]
@sebsoto It does currently, but while I was editing generate.sh I was wondering if it's ok that our community branches have separate csv descriptions than the community release ones. I see your point in it being a problem to update each time a community branch is created so I'm ok with closing it if you don't think it's necessary

**Pull request closed without merging**